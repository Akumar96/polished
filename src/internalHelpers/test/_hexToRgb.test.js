// @flow
import hexToRgb from '../_hexToRgb'

describe('transparentize', () => {
  it('it should correctly convert hex #fff to rgb', () => {
    expect(hexToRgb('#fff')).toMatchSnapshot()
  })
  it('it should correctly convert hex #000000 to rgb', () => {
    expect(hexToRgb('#000000')).toMatchSnapshot()
  })
  // it('it should throw an error on invalid color #00000', () => {
  //   expect(() => { hexToRgb('#00000') }).toThrow('Invalid hex color')
  // })
})
