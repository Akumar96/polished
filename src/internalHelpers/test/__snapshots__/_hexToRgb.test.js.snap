exports[`transparentize it should correctly convert hex #000000 to rgb 1`] = `"rgb(0, 0, 0)"`;

exports[`transparentize it should correctly convert hex #fff to rgb 1`] = `"rgb(255, 255, 255)"`;
